Sources/LibMobileCoin/Data16+withMcMutableBuffer.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+import LibMobileCoin
+
+extension Data16 {
```

Comment:
Not an error, but it would be nice if we could get rid of some of the repeated code in this type and Data64 and Data66. Maybe you could maintain type safety by creating a type for a buffer of a specified size and then have these three extend it but instantiate with a fixed size.

Reply:
Looked into doing this, and using a class I don't like because we loose the properties of a struct. Theres sort of way todo it with structs, but it would require I rewrite a bunch of other protocols and downgrade variable access levels which would end up being a high-impact change because these protocols are used widely in other places of the project. I think for now the duplication is acceptable. I can revisit in the future if I think of a third, better, option.